src/Modules/OrchardCore.Commerce/Services/ProductAttributeService.cs

```diff
@@ -50,6 +50,24 @@ public IEnumerable<ProductAttributeDescription> GetProductAttributeFields(Conten
             .Where(description => description.Field != null);
     }
 
+    public (ContentTypePartDefinition PartDefinition, ContentPartFieldDefinition FieldDefinition)
+        GetFieldDefinition(ContentTypeDefinition type, string attributeName)
+    {
+        var partAndField = attributeName.Split('.');
+        var partName = partAndField[0];
+        var fieldName = partAndField[1];
```

Comment:
We have an extension method for splitting a two-part string by a separator.
```suggestion
        var (partName, _, fieldName) = attributeName.Partition(".");
```